css/amp.css

```diff
@@ -78,6 +78,14 @@ body {
   right: 0;
 }
 
+.-amp-layout-flex-item {
```

Comment:
Let's add here `display: block` since you are documenting it explicitly below. I think this is what actually happens by default, but would be nicer to be explicit, I think.